doc/source/gitwash/development_workflow.rst

```diff
@@ -78,7 +78,7 @@ what the changes in the branch are for.  For example ``add-ability-to-fly``, or
     git fetch upstream
     # Make new feature branch starting at current trunk
     git branch my-new-feature upstream/main
-    git checkout my-new-feature
+	git checkout my-new-feature
```

Comment:
Woops, this introduces a TAB!

Reply:
Also, still a change in gitwash. We should contribute those fixes upstream at https://github.com/matthew-brett/gitwash !